gpustack/policies/candidate_selectors/sglang_resource_fit_selector.py

```diff
@@ -697,6 +753,203 @@ async def auto_select_multi_worker_multi_gpu_candidates(
 
         return []
 
+    async def _cal_mem_fraction_static(self, workers: List[Worker]):  # noqa: C901
+        """
+        Adapted from sglang's server_args memory fraction logic.
+
+        Args:
+            workers: List of workers used to determine GPU memory characteristics.
+        """
+        if self._selected_gpu_indexes_by_worker:
+            pass
```

Comment:
Why skip manual selected gpu here, since manual scheduling still rely on mem_fraction_static to calculate overcommit.